core/src/main/java/com/netease/arctic/data/DataTreeNode.java

```diff
@@ -107,6 +108,10 @@ public DataTreeNode parent() {
     return DataTreeNode.of(parentMask, parentIndex);
   }
 
+  public DataTreeNode copy() {
```

Comment:
As `DataTreeNode` is an Immutable class, I wonder why we need a copy method here.

Reply:
I will remove this method